sections/main-product.liquid

```diff
@@ -746,7 +747,7 @@
       }
     </script>
   </div>
-</section>
+</product-info>
```

Comment:
This is somehow breaking the padding applied and the content is flush with the header at the top: 
![](https://screenshot.click/24-07-pnum5-zx2mq.png)

Seems like it's missing a `display: block;` that it was getting from the user agent stylesheet: 
![](https://screenshot.click/24-11-ymvtp-xtpm3.png)

Reply:
Which should also fix the color scheme issue not working atm: 
![](https://screenshot.click/24-02-r0jh7-a3n9u.png)